lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -216,6 +216,18 @@ PYI_ESCAPE:
     end:
       targetState: END
 
+PYI_CRI_ESCAPE:
+  response:
+    type: page
+    pageId: pyi-cri-escape
+  events:
+    dcmaw:
+      targetState: CRI_DCMAW
```

Comment:
Also think we'll want to define a new state for this DCMAW journey so that it skips going to the address and fraud CRIs as the user will have already been through them